backend/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.2'
+
+services:
+  versewise-cms-backend:
+    build:
+      context: .
+      network: host
```

Comment:
Define a network at the bottom,  don't use host as a network 
- [ ] define a network